src/components/status-image/status-image.jsx

```diff
@@ -1,8 +1,16 @@
 import './status-image.css';
+import standing from '../../assets/standing-19@2x.png';
+import home from '../../assets/Home.svg';
+import campus from '../../assets/Wireframe.svg';
 
 function StatusImage(props) {
     return(
-        <div className="StatusImage">StatusImage</div>
+        <div className="StatusImage" id={props.available === false ? 'opacity40' : undefined}>
+            StatusImage
+            {!props.onCampus && <img src={home} alt="Home" />}
+            {props.onCampus && <img src={campus} alt="Campus" />}
```

Comment:
the conditional rendering is ok here but I wonder what would it be best in terms of performance.

we could assign the place in the render method

`const place = onCampus ? office : home;`

and then have only 1 <img> in the code and replace the src there.

`<img className="background" src={place} alt="stanting person"/>`

Again. Not sure what is the best for performance but I have the impression the code is more readable.